btree/btree.c

```diff
@@ -1574,11 +1593,20 @@ static int bnode_level(const struct nd *node)
 	return (node->n_type->nt_level(node));
 }
 
+#if 0
```

Comment:
Remove this function if we do not need it.

Reply:
Done.